shared/chat/conversation/messages/message-popup/attachment/index.js

```diff
@@ -50,6 +51,7 @@ const AttachmentPopupMenu = (props: Props) => {
       ? [{disabled: props.pending, onClick: props.onShareAttachment, title: 'Share'}]
       : []),
     ...(props.onDownload ? [{disabled: props.pending, onClick: props.onDownload, title: 'Download'}] : []),
+    ...(isMobile ? [{onClick: props.onAddReaction, title: 'Add a reaction'}] : []),
```

Comment:
really should be controlled by the connector. then have the check be props.onAddReaction (like onDownload above)